### PR TITLE
Dataset "GSHHG Earth distance to shoreline": Update docs and add coastlines to images

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
_Please see comment https://github.com/GenericMappingTools/pygmt/pull/3706#issuecomment-2558768025 for context_.

Update the documentation for the remote dataset "GSHHG Earth distance to shoreline":

- [ ] Mention distance unit is km
- [ ] Mention distances have both negative and positive values. Positive means land to coastline and negative means ocean to coastline.
- [ ] Add the coastline to the image to make the it more readable.

**Preview**: https://remote-datasets--126.org.readthedocs.build/en/126/earth-dist.html

### CODE REVIEW

File: scripts/GMT_earth_maps.sh [hunk 1]
```diff
@@ -8,10 +8,13 @@ gmt set GMT_DATA_SERVER candidate
 let k=0
 I=""
 for set in ${list[@]}; do
-	gmt begin GMT_earth_${set} jpg
+    gmt begin GMT_earth_${set} jpg
         gmt grdimage @earth_${set}_10m -JQ0/7.5c -Rd ${I}
+        if [ set = dist ]; then
+          gmt coast -W0.01p,white
```
Comment (seisman) [inline scripts/GMT_earth_maps.sh:14]:
```suggestion
          gmt coast -W1/0.2p,white -Df
```
1. Need to plot coastlines only, no lakes. 
2. Need to use the full resolution GSHHG data to show the small-area islands, to match the zero-values in the ocean.

Comment (yvonnefroehlich) [reply scripts/GMT_earth_maps.sh]: Updated the images in commit https://github.com/GenericMappingTools/remote-datasets/pull/126/commits/f857778c36b2f8e1ae9fc1d0b4918c8cde414c1d.

**Comment by seisman** [comment]
Looks good to me and ping @Esteban82 for review.

**Review by Esteban82 — APPROVED** [review]

**Comment by Esteban82** [comment]
It looks fine @yvonnefroehlich. Is still a WIP? If not, please remove it from the title.

**Comment by yvonnefroehlich** [comment]
> It looks fine @yvonnefroehlich. Is still a WIP? If not, please remove it from the title.

This PR should be finished. I updated the title.

**Pull request merged**